.github/pull_request_template.md

```diff
@@ -1,6 +1,3 @@
-This is the link to the project: [TITLE](URL)
-
-- [ ] **I have read the [contributing guidelines](contributing.md).**
```

Comment:
I would also advocate for this line staying in for two reasons:
1. It makes people re-read the rules. Making sure they have read and understood them all by making them read the rules twice.
2. While atm guidelines are the same as the PR template we might expand upon the guidelines in the future. Giving more explanations for some guidelines or pointers to resources.

Reply:
If we want people to re-read rules, then the rules should be 2x shorter to compensate. Having a checklist already makes me consciously think through each one.

If we want the guidelines to be different, then there should not be any duplicate item. The way it was before, I found it to be a rough, convoluted experience to submit PRs, and I was definitely less thorough because of it